sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawExternalVolume.java

```diff
@@ -0,0 +1,39 @@
+package com.mesosphere.sdk.specification.yaml;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * RawHostVolume YAML spec.
+ */
+public final class RawExternalVolume {
+
+  private final String containerPath;
+
+  private final String mode;
+
+  private final int size;
+
+  private RawExternalVolume(
+          @JsonProperty("size") int size,
```

Comment:
```suggestion
      @JsonProperty("size") int size,
```
(nit) extra indentation.